docs/Show.md

```diff
@@ -157,46 +157,3 @@ export const PostShow = (props) => (
     </Show>
 );
 ```
-
-## Declaring Fields at Runtime
-
-You might want to dynamically define the fields when the `<Show>` component is rendered. It accepts a function as its child, and this function can return a Promise. If you defined an `authClient` on the `<Admin>` component, the function child of the `<Show>` component will receive the result of a call to `authClient` with the `AUTH_GET_PERMISSIONS` type (you can read more about this in the [Authorization](./Authorization.md) chapter).
```

Comment:
Why did you remove it?

Reply:
Same reason, child as a function is not supported anymore